testgranularity/systemtesting.md

```diff
@@ -1,184 +1,36 @@
-```
- 
- * Selenium Webdriver and Robot Framework:
- 
-    Open Browser https://www.facebook.com/         firefox
-
-
-## CODE STRUCTURE
-
-The structure of the script is simple and can be divided into four sections:
-
-### Documentation
-This keyword precedes the description of the functionality or test scenario. Be aware, because if the documentation is made in the section **Settings**, it should not be enclosed in square brackets. They are only used within **Test Cases**
-
-`This test open the webpage “https://www.amazon.in”
-Search for “unit test book” and then verify if the results match.`
-
-### Settings
-This is where you configure the libraries to be used, paths to auxiliary files (page objects, for example), contexts and hooks.
-
-`Library           Selenium2Library
-Suite Setup  Go to homepage
-Suite Teardown  Close All Browsers`
-
-### Variables
-List of variables to be used (preferably with description) and definition of the values of some of these variables.
-
-`${HOMEPAGE}      https://www.amazon.in
-${BROWSER}        chrome`
-
-### Test Cases
-This is the most important section, because without it your test will not run. This is where the test cases / scenarios are, with or without implementation.
-
-`Amazon page and find books
-    Amazon and check results   unit test book   results for "unit test book"`
-
-### Keywords
-
-Here you can define keywords or implement the test scenarios described above. All of the above sections are optional, depending on how your code was written, except for **Test Cases**. I advise you to always use them for a better organization of the code. 
-The labels below are not mandatory, but they also help with the organization.
-
-`Amazon and check results`
-
-`    [Arguments]     ${searchkey}     ${result}`
-
-`    Input Text      id=twotabsearchtextbox     ${searchkey}`
-
-`    Click Element    xpath=//div[@class='nav-search-submit nav-sprite']/input[@class='nav-input' and 1]`
-
-`    Wait Until Page Contains    ${result}`
- 
-`Go to homepage
-    Open Browser    ${HOMEPAGE}     ${BROWSER}`
-    
-### Spaces and Variables
-The variables in the Robot are represented by `${variable}`. The Robot has the peculiarity of ignoring a space between words. 
-`${variable_name}` is equal to `${variable name}`. They are also case insensitive. The arguments are divided into at least two spaces. For example:
-
-* command (two spaces) argument1 (two spaces) argument 2.
-
-Note that the space in “argument 2” is ignored. The assignment sign (=) is also optional. Values can be assigned, either:
-* `$ {value} value`
-* `$ {value} = value`
-
-An important observation to be made is that, being the Robot Framework based on Python, it is worth mentioning that this is an indented language, that is, command blocks are separated by spaces or tabs, forming a mandatory visual indentation. There are no *“open”* and *“close”* symbols, Robot inherits this formatting.
+![Img](../assets/systemtesting_diagram.png)
```

Comment:
It would be better to change the caption from `Img` to something more meaningful.